pkg/bootstrap/types.go

```diff
@@ -41,15 +40,24 @@ type OVNConfigBoostrapResult struct {
 	HyperShiftConfig *OVNHyperShiftBootstrapResult
 }
 
+type OVNUpdateStatus struct {
+	Kind         string
+	Namespace    string
+	Name         string
+	Version      string
+	IPFamilyMode string
+	Progressing  bool
+}
+
 type OVNBootstrapResult struct {
-	MasterAddresses         []string
-	ClusterInitiator        string
-	ExistingMasterDaemonset *appsv1.DaemonSet
-	ExistingNodeDaemonset   *appsv1.DaemonSet
-	ExistingIPsecDaemonset  *appsv1.DaemonSet
-	OVNKubernetesConfig     *OVNConfigBoostrapResult
-	PrePullerDaemonset      *appsv1.DaemonSet
-	FlowsConfig             *FlowsConfig
+	MasterAddresses       []string
+	ClusterInitiator      string
+	MasterUpdateStatus    *OVNUpdateStatus
```

Comment:
Can you add some comments about what all of these are? In other words, which resources they correspond to?